wire-library/wire-runtime-swift/src/main/swift/propertyWrappers/StringEncodedValues.swift

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/// Converts an array of values to/from their string equivalent when serializing with Codable.
+@propertyWrapper
+public struct StringEncodedValues<Value: StringCodable> {
```

Comment:
This would be trivial to support arbitrary collections (like Set) with a minimal additional protocol that enforced an `init(_ sequence)` (which both types support)

Reply:
- [ ] Add set support